src/mscorlib/shared/System/IO/PathInternal.Unix.cs

```diff
@@ -23,6 +24,11 @@ internal static partial class PathInternal
         internal const string ParentDirectoryPrefix = @"../";
 
         internal static int GetRootLength(string path)
+        {
```

Comment:
@jkotas @danmosemsft  do we need to keep these internal string based apis ?

Reply:
```
internal static bool IsPartiallyQualified(string path)
```
can i remove these 2 apis ?

Reply:
Yes, you can remove these.